lib/subscribers/mysql/connection-query2.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const MySQLConnectionQuerySubscriber = require('./connection-query')
```

Comment:
i'm not a fan of attaching 2 to these. After looking at this I think it would make more sense to do:

 * drop `connection-` from all the names. 
 * replace files with `2` with `mysql2-<name>`